Homework 23-01-17 - Functions.playground/Contents.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+
+//Functions excutes code. Some functions define arguments that you can use to pass in data to help the functions do it work.
+
+//A function is like a little machine.
+
+//Call a functions by name functions and that ()
+
+//A argument is the valeu a caller gives to a function's parameter.
+
+
+                                //DEFINING A FUNCtIONS//
+
+func printGreeting() {
+    print("Hello, playground.")
+}
+printGreeting()
+
+
+                                //FUNCTION PARAMETERS//
+
+//Give functions a extra life whit a Parameter(s). See (name: "Jeroen")
+
+func printPersonalGreeting(name: String) {
+    print("Hello \(name), welcome to your playground.")
+}
+    printPersonalGreeting(name: "Jeroen")
+
```

Comment:
Why is printPersonalGreeting(name: "Jeroen") indented in line with print inside of  printPersonalGreeting function, should indent to left , it is out side of function